src/HookGen/README.md

```diff
@@ -0,0 +1,23 @@
+# HookGen (WIP)
+
+This is an experimental development tool that dumps stack trace for each function call. It's uses
+dynamic analysis (at runtime) and possibly static analysis in future to combine best of both.
+
+## Building
+
+```
+phpize
+./configure
+make
+make install
+```
+
+## Usage
+
+php -dextension=hookgen.so [php app]
```

Comment:
I've seen this named `hookgen` and `php_auto_instr_cfg`, maybe should have the same name everywhere for consistency?

Reply:
True, another leftover.